### PR TITLE
Additional line for Gems popover heading for better support in translations

### PR DESCRIPTION
**Pull request by ghost**
https://github.com/HabitRPG/habitrpg/issues/2932

### CODE REVIEW

File: views/options/social/challenges.jade [hunk 1]
```diff
@@ -145,8 +145,7 @@ script(type='text/ng-template', id='partials/options.social.challenges.html')
                   =env.t('by') + ' '
                   strong {{challenge.leader.profile.name}}
                 li
-                  | {{challenge.memberCount}}
-                  = ' ' + env.t('participants')
+                  =env.t('participants', {membercount: "{{challenge.memberCount}}"})
```
Comment (crookedneighbor) [inline views/options/social/challenges.jade:148]: Am I missing something? Doesn't this require a change to `common/locales/en/challenge.json`?

Comment (Alys) [inline views/options/social/challenges.jade:148]: See this: https://github.com/HabitRPG/habitrpg-shared/pull/162

**Comment by crookedneighbor** [comment]
@GitHubSphinx I've merged this in, but since [I needed to do a rebase and pull in your changes from shared](https://github.com/HabitRPG/habitrpg/commit/7efb5948382fe792e7d753b155be64c6c7b72322), it didn't automatically merge in this PR (so I'm closing it now). Since the `gemsPopTitle` is a duplicate of `gems` (both resulting in "Gems"), I ended up reverting that change and just accepted the challenges string change.

It's possible that you were referring to a request on Github or Trello or something to separate those out, and I just missed it, or maybe you had an awesome reason for doing that. If that's the case, please speak up. :smile:

What's your `uuid`? I'll notate this towards your next contributor tier.

**Comment by crookedneighbor** [comment]
Aaaaaaaaand I just re-read the issue you were referring to originally. You do want separate strings. I'll put it back!

**Comment by crookedneighbor** [comment]
That's what I get for not reading.

**Comment by ghost** [comment]
Dear @crookedneighbor, dear @Alys !
Thank you for regarding the issue. 
I'm not sure about the outcome, as I've been away for about a year and I've missed all the fun. I've made the request quite a long time ago. If I remember right, these pull requests were just to test if I can make some small things regarding translation issues myself.
As I'm back, but haven't got much experience with github, could you please let me know if I can try to do such small things here and there or - if I'm wrong, should I better stay away from github?
BTW my uid is ffa26f90-f4cf-40b5-8b3b-aca988ab8ed2 if you'd like to find me on HRPG.

**Pull request closed without merging**